bundler/lib/dependabot/bundler/native_helpers.rb

```diff
@@ -1,10 +1,46 @@
 # frozen_string_literal: true
 
+require "bundler"
+require "dependabot/shared_helpers"
+
 module Dependabot
   module Bundler
     module NativeHelpers
-      def self.helper_path
-        "bundle exec ruby #{File.join(native_helpers_root, 'run.rb')}"
+      def self.run_bundler_subprocess(function:, args:, bundler_version:)
+        # Run helper suprocess with all bundler-related ENV variables removed
+        with_unbundled_env do
+          SharedHelpers.run_helper_subprocess(
+            command: helper_path(bundler_version: bundler_version),
+            function: function,
+            args: args,
+            env: {
+              # Bundler will pick the matching installed major version
+              "BUNDLER_VERSION" => bundler_version,
+              "BUNDLE_PATH" => ".bundle/"
```

Comment:
Possible follow on exploration, I wonder if we should investigate setting `BUNDLE_GLOBAL_GEM_CACHE=true`, not sure it will do anything but we do resolve things multiple times during a single dependency update so might speed things up 🤔

Reply:
Yeah, I think that's a good call 👍🏻

Reply:
FWIW, the use of `.bundle/` is not heavily opinionated, it was just a wild ass guess at a good starting point.